admin_guide/high_availability.adoc

```diff
@@ -256,52 +256,22 @@ service account created previously or create a new *ipfailover* service account.
 Ensure that either the *router* service account exists as described in
 link:../install_config/install/deploy_router.html[Deploying a Router] or create
 a new *ipfailover* service account. The example below creates a new service
-account with the name *ipfailover*:
+account with the name *ipfailover* in the *default* namespace:
 +
 ====
 ----
-$ echo '
-  { "kind": "ServiceAccount",
-    "apiVersion": "v1",
-    "metadata": { "name": "ipfailover" }
-  }
-  ' | oc create -f -
+$ oc create serviceaccount ipfailover -n default
 ----
 ====
 
-. You can manually edit the *privileged* SCC and add the
-*ipfailover* service account, or you can script editing the *privileged* SCC if
-you have `jq` installed.
+. Add the *ipfailover* service account to the *privileged* SCC
 
-.. To manually edit the *privileged* SCC, run:
+.. To add the *ipfailover* service account in the *default* namespace
```

Comment:
This doesn't need to have a substep (`..`) anymore since there's only one method now.